packages/stable/src/__tests__/api/documentsApi.unit.spec.ts

```diff
@@ -126,32 +126,7 @@ describe('Documents unit test', () => {
               item: {
                 geoLocation: {
                   type: 'MultiPolygon',
-                  coordinates: [
-                    [
-                      [
-                        [40.0, 40.0],
-                        [20.0, 45.0],
-                        [45.0, 30.0],
-                        [40.0, 40.0],
-                      ],
-                    ],
-                    [
-                      [
-                        [20.0, 35.0],
-                        [10.0, 30.0],
-                        [10.0, 10.0],
-                        [30.0, 5.0],
-                        [45.0, 20.0],
-                        [20.0, 35.0],
-                      ],
-                      [
-                        [30.0, 20.0],
-                        [20.0, 15.0],
-                        [20.0, 25.0],
-                        [30.0, 20.0],
-                      ],
-                    ],
-                  ],
+                  coordinates: [10, 20],
```

Comment:
The types no longer support this format. Is this expected? @tommythorsen

Reply:
No, this is not quite right. We probably screwed up in this PR https://github.com/cognitedata/service-contracts/pull/2208. This is probably blocking this PR, but could you just comment out the test instead of changing it?

Reply:
Done!
It indeed blocks this PR. When do you think we could look at fixing the service contract?

Reply:
This would currently incur a breaking change it seems. which is fine if it is, but it sounds like its unintentional. so we'd need to fix up the service contracts and redo the openapi import before
<img width="684" alt="image" src="https://github.com/cognitedata/cognite-sdk-js/assets/38425820/c97fd063-23ec-4f80-9c23-d3ba11b54a44">